src/OpenSpaceToolkit/Astrodynamics/Dynamics/Tabulated.cpp

```diff
@@ -40,12 +40,12 @@ Tabulated::Tabulated(
 
     if (aWriteCoordinateSubsets
             .map<Index>(
-                [](const auto& coordinatesSubset)
+                [](const auto& coordinateSubset)
                 {
-                    return coordinatesSubset->getSize();
+                    return coordinateSubset->getSize();
                 }
             )
-            .reduce(std::plus<Index>()) != (Index)aContributionProfile.cols())
+            .reduce(std::plus<Index>()) != static_cast<Index>(aContributionProfile.cols()))
```

Comment:
For my knowledge, why is a static cast necessary here?

Reply:
Because its safer and more "searchable" and expresses the programmer's intention more clearly than a c-style cast

https://stackoverflow.com/questions/1609163/what-is-the-difference-between-static-cast-and-c-style-casting
https://www.reddit.com/r/cpp_questions/comments/f26n5x/why_use_static_cast_over_c_style_cast_for/